### DISCUSSION TITLE
Option for folder names on Dashboard lists

### DISCUSSION THREAD

**Discussion by sanchomuzax** [grafana/grafana#38158] [Dashboards, Panels & Visualizations]
**What would you like to be added**: Please, add an option for Dashboard list panel which allows to turn off the folder name labels. (These tags are new, released in version 7.2.)

![image](https://user-images.githubusercontent.com/2911588/94401576-1a577600-016b-11eb-8d30-cdd97958d1de.png)

**Why is this needed**: Displaying the label carries unnecessary information and increases the height of the box.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by huynhsamha** [comment]
I noticed there have been many issues related to this feature, such as #32115, #29814, and #27849.

I also find that displaying the same folder name for each item in the dashboard list is quite annoying, loses a lot of space, and is distracting, since the panel itself already specifies the folder name.

![grafana-dashlist-merge-edited](https://user-images.githubusercontent.com/24195776/137619208-8cc6f3a3-af4d-4044-a04f-ebfb9a4c5c45.jpg)

I hope Grafana team can add this display is optional.

Thanks team.

**Comment by tarko** [comment]
Yes please, folder names waste a lot of vertical space

**Comment by carlosmacapuna** [comment]
+1

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82955. Feel free to continue the discussion around this there. Thank you!
